devfiles/java-maven/devfile.yaml

```diff
@@ -18,8 +18,6 @@ components:
     alias: maven
     image: quay.io/eclipse/che-java11-maven:nightly
     env:
-      - name: MAVEN_CONFIG
-        value: /home/user/.m2
```

Comment:
I think we should also unset `MAVEN_CONFIG` here, since the base image has `MAVEN_CONFIG=/root/.m2` by default. Our sample project doesn't use `mvnw` but it might trip up someone trying to import their own project.

Reply:
Unset for all maven based devfiles